Makefile

```diff
@@ -131,6 +131,15 @@ remove: ## Remove all containers
 	$(info Remove all containers)
 	@docker-compose rm -sv
 
+renew: ## Refresh all relevant services that were impacted by prior Pull Request.
```

Comment:
A new command we can update when we need to update specific services based on the prior PR.  It's not a perfect solution because if you fall behind in running it, you'll have to manually perform the necessary refreshes.